packages/remix-server-runtime/__tests__/server-test.ts

```diff
@@ -444,8 +448,10 @@ describe("shared server runtime", () => {
       });
 
       let result = await handler(request);
+      let data = await result.json();
       expect(result.status).toBe(500);
-      expect((await result.json()).message).toBe("Unexpected Server Error");
+      expect(data.name).toBe("Error");
```

Comment:
shouldn't it be "CustomError" here ?

Reply:
These tests are checking that the error name is redacted in production mode, I think they are correct as is.

Reply:
ah yes, you're right,  I've been fooled by the name of the config `ServerMode.Test`

Reply:
Really strange that all tests are passing although this isn't correct 🤔

Reply:
@MichaelDeBoey do you still see something wrong here? 🤔

Reply:
We're still checking for `Error` instead of `customError`, so yes I guess there's still something wrong

Reply:
It seems that `Error` is expected here, see comment above:

> These tests are checking that the error name is redacted in production mode, I think they are correct as is.

Reply:
@jnicklas So why are we using `CustomError` in our tests if we're never checking anything from them? 🤔

Reply:
There is `deserializeError` method too, if you don't handle that one it still will return Error instead of CustomError.

https://github.com/remix-run/remix/blob/fd00bc657ef7f2b55c53e5f9dbd77185fa7d5734/packages/remix-react/components.tsx#L224